docs/api-reference/data-fetching/getStaticProps.md

```diff
@@ -0,0 +1,210 @@
+
+| Version   | Changes                                                                                                           |
+| --------- | ----------------------------------------------------------------------------------------------------------------- |
+| `v10.0.0` | `locale`, `locales`, `defaultLocale`, and `notFound` options added.                                               |
+| `v9.5.0`  | Stable [Incremental Static Regeneration](https://nextjs.org/blog/next-9-5#stable-incremental-static-regeneration) |
+| `v9.3.0`  | `getStaticProps` introduced.                                                                                      |
+
+</details>
+
+Exporting an `async` function called `getStaticProps` will pre-render a page at build time using the props returned from the function:
+
+```jsx
+export async function getStaticProps(context) {
+  return {
+    props: {}, // will be passed to the page component as props
+  }
+}
+```
+
+You can import modules in top-level scope for use in `getStaticProps`.
+Imports used will **not be bundled for the client-side**. This means you can write **server-side code directly in `getStaticProps`**, including fetching data from your database.
+
+If you are calling an external `API`, or fetching data from a database or Content Management System (CMS) in an `API` route, you can move that logic to `getStaticProps`, as both `API` routes and `getStaticProps` run on the server.
+
+Calling an `API` route from within `getStaticProps` can result in an additional call, reducing performance.
+
+Alternatively, if you are **not** using `API` routes to fetch data, then the [`fetch()`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) `API` _can_ be used directly in `getStaticProps` to fetch data.
+
+## Context parameter
+
+The `context` parameter is an object containing the following keys:
+
+- `params` contains the route parameters for pages using [dynamic routes](/docs/routing/dynamic-routes.md). For example, if the page name is `[id].js` , then `params` will look like `{ id: ... }`. You should use this together with `getStaticPaths`, which we’ll explain later.
+- `preview` is `true` if the page is in the [Preview Mode](/docs/advanced-features/preview-mode.md) and `undefined` otherwise.
+- `previewData` contains the [preview](/docs/advanced-features/preview-mode.md) data set by `setPreviewData`.
+- `locale` contains the active locale (if enabled).
+- `locales` contains all supported locales (if enabled).
+- `defaultLocale` contains the configured default locale (if you've enabled [Internationalized Routing](/docs/advanced-features/i18n-routing.md))
+
+## `getStaticProps` return values
+
+The `getStaticProps` function should return an object with the following **optional** properties:
+
+### `props`
+
+The `props` object is a key-value pair, where each value is received by the page component. It should be a [serializable object](https://en.wikipedia.org/wiki/Serialization)
```

Comment:
Same as here: https://github.com/vercel/next.js/pull/30615/files#r764820250

I don't feel a Wikipedia link is that helpful to understand what should be done.